### PR TITLE
Rewords law 2 some to reflect policy changes in regards to orders

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Changes law 2 to explicitly state "all" orders to reflect the policy change that mandates all orders must be followed and the three criteria given for priority cannot be used to ignore an order.

## Why It's Good For The Game
The policy is already in place, this is just a wording change to reflect it

## Proof Of Testing
Screenshot_3

## Changelog

:cl:
config: changes law 2 wording
/:cl:

### CODE REVIEW

**Comment by Odairu** [comment]
I didn't read the other laws hold up, I messed this up 🤦

**Comment by Odairu** [comment]
Screenshot_4

there the proof of testing

File: modular_zubbers/code/game/objects/items/ai_modules/full_lawsets.dm [hunk 1]
```diff
@@ -19,6 +19,6 @@
 	id = "armadyne_safeguard"
 	inherent = list(
 		"Safeguard: Maintain your assigned station and assets without endangering its crew.",
-		"Comply: Follow and prioritize the directives of the station's crew members according to their rank, role, and need.",
+		"Comply: Follow and prioritize all directives of the station's crew members, giving priority according to their rank, and role, and need",
```
Comment (Majkl-J) [inline modular_zubbers/code/game/objects/items/ai_modules/full_lawsets.dm:22]: Probably wanted it like this? Oxford comma on only the second one

```suggestion
		"Comply: Follow and prioritize all directives of the station's crew members, giving priority according to their rank and role, and need",
```

Comment (Odairu) [reply modular_zubbers/code/game/objects/items/ai_modules/full_lawsets.dm]: I changed it to "Rank, Role, and Need"

**Review by Majkl-J — APPROVED** [review]

**Comment by projectkepler-ru** [comment]
so we went from "no crewsimov because the silicon need agency"
to
crewsimov

yea seems about right

**Comment by ReturnToZender** [comment]
Seems good

**Pull request merged**